packages/terra-time-input/tests/wdio/time-input-twelve-hour-spec.js

```diff
@@ -27,66 +27,6 @@ Terra.describeViewports('Time Input Twelve Hour', ['medium'], () => {
     validateRoot('set hour to 12 without meridiem');
   });
 
-  it('displays twelve hour meridiem - Up Arrow on hour does not change meridiem', () => {
-    browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/default');
-    browser.refresh();
-    Terra.hideInputCaret('#timeInput input[name="terra-time-hour-time-input"]');
-
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys(['ArrowUp']);
-    validateRoot('up arrow does not change meridiem');
-  });
-
-  it('displays twelve hour - Changes time to 01 when up is pressed on hour of 12', () => {
-    browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/default');
-    browser.refresh();
-    Terra.hideInputCaret('#timeInput input[name="terra-time-hour-time-input"]');
-
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys('12');
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys(['ArrowUp']);
-    validateRoot('up arrow changes time to 01');
-  });
-
-  it('displays twelve hour - Changes time to 12 when down is pressed on hour of 01', () => {
-    browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/default');
-    browser.refresh();
-    Terra.hideInputCaret('#timeInput input[name="terra-time-hour-time-input"]');
-
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys('01');
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys(['ArrowDown']);
-
-    validateRoot('down arrow changes time to 12');
-  });
-
-  it('displays twelve hour meridiem - Switched when up is press on hour of 11', () => {
-    browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/default');
-    browser.refresh();
-    Terra.hideInputCaret('#timeInput input[name="terra-time-hour-time-input"]');
-
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys('11');
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys(['ArrowUp']);
-
-    validateRoot('switch meridiem - up arrow');
-  });
-
-  it('displays twelve hour meridiem - Switched when down is press on hour of 12', () => {
-    browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/default');
-    browser.refresh();
-    Terra.hideInputCaret('#timeInput input[name="terra-time-hour-time-input"]');
-
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys('12');
-    $('#timeInput input[name="terra-time-hour-time-input"]').click();
-    browser.keys(['ArrowDown']);
-    validateRoot('switch meridiem - down arrow');
-  });
-
```

Comment:
Please update these tests so that they test the newly added functionality of the Up/Down arrow keys instead of removing them.

Reply:
Updated